core/scripts/keystone/src/03_gen_crib_cluster_overrides_cmd.go

```diff
@@ -23,7 +23,11 @@ func (g *generateCribClusterOverrides) Name() string {
 func (g *generateCribClusterOverrides) Run(args []string) {
 	fs := flag.NewFlagSet(g.Name(), flag.ContinueOnError)
 	chainID := fs.Int64("chainid", 11155111, "chain id")
-	outputPath := fs.String("outpath", "../crib", "the path to output the generated overrides")
+	cribRepoPath := os.Getenv("KEYSTONE_CRIB_REPO_PATH")
+	if cribRepoPath == "" {
+		cribRepoPath = "../../crib"
```

Comment:
Just to confirm, this should be the repo path for CRIB, right?

Reply:
Sounds like it is per @HenryNguyen5.

Reply:
Yeah, think the only reasonable default would now be `../../../crib`, assuming that's the repo name and they're cloning it right beside the core repo. I would definitely add an existence check now since the repo path is going to vary between machines

Reply:
Thanks, updated in 7ecb9da536302d4532b83e09ab27b47389ac6454.